manim_slides/present.py

```diff
@@ -562,7 +562,15 @@ def __init__(
         self.thread.start()
 
     def keyPressEvent(self, event):
+        self.config = Config()
```

Comment:
```suggestion
```

Here, you define a default config for every key press. That is quite inefficient and does not use the config that was passed to `self.thread`. See above

Reply:
Edit: can you modify `Display.__init__` to take `config: Config = Config()` instead of just `config`? Then, also add `config: Config = Config()` to `App.__init__` arguments, and modify the line `self.thread = Display(*args, **kwargs)` to be `self.thread = Display(*args, config=config, **kwargs)`.